src/debugging/coreclr/CommandLineDotNetClient.ts

```diff
@@ -48,6 +58,60 @@ export class CommandLineDotNetClient implements DotNetClient {
             return undefined;
         }
     }
+
+    public async isCertificateTrusted(): Promise<boolean | undefined> {
```

Comment:
We usually stay away from returning results which can also be undefined. It requires a further check at the calling function. It can also be coerced into a false.

Using an enum could help. I'll defer to Eric on this though.

Reply:
I'll do an enum here, it's easy enough.